### ISSUE TITLE
NTR: Mycobacterium present in CSF culture (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7906]
New term request Mycobacterium present in CSF culture (samant_101122005934): Cerebrospinal fluid  UMLS: 2026564

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000835
- label: Positive CSF mycobacterial culture
- parents: [Positive CSF infectious agent test (HP:0430123)](https://hpo.jax.org/app/browse/term/HP:0430123)
- pmid: 
- def: Growth of mycobacteria in a culture derived from cerebrospinal fluid (CSF).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7906
- synonyms: Positive CSF mycobacterium culture(NONE)

**Comment by MickeySegal** [comment]
We already discussed this in https://github.com/obophenotype/human-phenotype-ontology/issues/8059 and you assigned a different HPO code so this one seems redundant.

**Comment by pnrobinson** [comment]
@MickeySegal
The following is correct:
Posiitive mycobacterium CSF culture HP:6000543
Please disregard the HP:6000835, that was an error.
